modAionBase/test/org/aion/base/util/ByteUtilTest.java

```diff
@@ -0,0 +1,449 @@
+package org.aion.base.util;
+
+import org.junit.Test;
+
+import javax.sound.midi.SysexMessage;
+import java.math.BigInteger;
+import java.util.Set;
+
+import static org.aion.base.util.ByteUtil.*;
+import static org.junit.Assert.*;
+
+public class ByteUtilTest {
```

Comment:
Instead of looping through the arrays you can use the lib JUnitParams-1.1.1.jar and wite parametrized tests.

Reply:
sure